lib/capybara/webkit/node.rb

```diff
@@ -47,6 +47,19 @@ def set(value)
       invoke "set", *[value].flatten
     end
 
+    def send_keys(*keys)
+      invoke("sendKeys", [keys].flatten.map { |key|
```

Comment:
I'm following https://github.com/bbatsov/ruby-style-guide/issues/162 here, is that OK with you?